app.js

```diff
@@ -0,0 +1,28 @@
+require('dotenv').config();
+const express = require('express');
+const cors = require('cors');
+const mongoose = require("mongoose");
+const app = express();
+
+const apiRoutes = require("./src/modules/routes/routes")
+
+app.use(cors());
+
+app.use(express.json());
+app.use("/", apiRoutes);
+const URL = process.env.URL;
+
+const launch = () => {
+  try {
+    mongoose.connect(URL, { useNewUrlParser: true, useUnifiedTopology: true });
```

Comment:
you don't have to create a separate variable, but just write it. It is only used once anyway 
mongoose.connect(process.env.URL,